x-pack/plugins/reporting/server/export_types/png/execute_job/index.ts

```diff
@@ -36,7 +36,7 @@ export const runTaskFnFactory: QueuedPngExecutorFactory = function executeJobFac
     const generatePngObservable = await generatePngObservableFactory(reporting);
     const jobLogger = logger.clone([jobId]);
     const process$: Rx.Observable<TaskRunResult> = Rx.of(1).pipe(
-      mergeMap(() => decryptJobHeaders({ encryptionKey, job, logger })),
+      mergeMap(() => decryptJobHeaders(encryptionKey, job.headers, logger)),
       map((decryptedHeaders) => omitBlacklistedHeaders({ job, decryptedHeaders })),
       map((filteredHeaders) => getConditionalHeaders({ config, job, filteredHeaders })),
```

Comment:
I plan to have a later PR that makes all the common helper functions consistent and not take a single options parameter. 

The general direction of the evolution of these functions is that they need fewer / smaller objects over time.